datafusion/optimizer/src/eliminate_cross_join.rs

```diff
@@ -47,81 +48,93 @@ impl EliminateCrossJoin {
 /// For above queries, the join predicate is available in filters and they are moved to
 /// join nodes appropriately
 /// This fix helps to improve the performance of TPCH Q19. issue#78
```

Comment:
This current doc only has equality predicates example, maybe update it with this change?